CONTRIBUTING.md

```diff
@@ -100,6 +100,12 @@ Enable Corepack, which installs Yarn automatically:
 corepack enable
 ```
 
+Build the API reference:
+
+```sh
+./build_api_reference.sh
```

Comment:
```suggestion
make build-api-reference
```

Reply:
it would not work as we are in website/ directory

Reply:
Huh. Do we need to be there?